app/errors.py

```diff
@@ -0,0 +1,30 @@
+class VaccineError(Exception):
+    """The exception is raised
+    when there is some problems with vaccination"""
+
+
+class NotVaccinatedError(VaccineError):
+    def __init__(
+            self,
+            message: str = "Vaccine is obligatory!"
+    ) -> None:
+        self.message = message
+        super().__init__(message)
```

Comment:
super is referring to parent class init, but there`s no init in VaccineError class